dbt_subprojects/daily_spellbook/macros/project/lifi/lifi_extract_bridge_data_macro.sql

```diff
@@ -0,0 +1,37 @@
+{% macro lifi_extract_bridge_data(blockchain) %}
+
+{% set bridge_data_fields = [
+    'transactionId',
+    'bridge',
+    'integrator',
+    'referrer',
+    'sendingAssetId',
+    'receiver',
+    'minAmount',
+    'destinationChainId'
+] %}
+
+select
+    contract_address,
+    evt_tx_hash as tx_hash,
+    evt_index,
+    evt_block_time as block_time,
+    evt_block_number as block_number,
+    cast(date_trunc('day', evt_block_time) as date) as block_date,
+    {% for field in bridge_data_fields %}
+    {% if field in ['transactionId', 'referrer', 'sendingAssetId', 'receiver'] %}
+    from_hex(json_extract_scalar(bridgedata, '$.{{ field }}')) as {{ field }},
+    {% elif field == 'minamount' %}
+    cast(json_extract_scalar(bridgedata, '$.{{ field }}') as double) as {{ field }},
+    {% else %}
+    json_extract_scalar(bridgedata, '$.{{ field }}') as {{ field }},
+    {% endif %}
+    {% endfor %}
+    '{{ blockchain }}' as source_chain,
+    {{ dbt_utils.generate_surrogate_key(['evt_tx_hash', 'evt_index']) }} as transfer_id
+from {{ source('lifi_' ~ blockchain, 'lifidiamond_v2_evt_lifitransferstarted') }}
```

Comment:
somewhere along the way, upper/lower case matters for source tables. looks like your latest commit changed this to all lowercase, while the source yml file defined in the PR uses camelcase. your best bet is to put this back to camelcase to match source file (or other way around).

tbh i haven't confirmed which step casing matters, but i'm fairly certain it's this. i don't think the dunesql engine cares, if you run on the app with either i believe it works.